app/backend/app.py

```diff
@@ -75,50 +95,53 @@ def static_file(path):
 # Serve content files from blob storage from within the app to keep the example self-contained. 
 # *** NOTE *** this assumes that the content files are public, or at least that all users of the app
 # can access all the files. This is also slow and memory hungry.
-@app.route("/content/<path>")
-def content_file(path):
+@app.route("/", defaults={"path": "index.html"})  
+@app.route("/<path:path>")  
+def static_file(path):  
+    return app.send_static_file(path)  
+  
+@app.route("/content/<path>")  
+def content_file(path):  
     blob = blob_container.get_blob_client(path).download_blob()
     if not blob.properties or not blob.properties.has_key("content_settings"):
-        abort(404)
-    mime_type = blob.properties["content_settings"]["content_type"]
-    if mime_type == "application/octet-stream":
-        mime_type = mimetypes.guess_type(path)[0] or "application/octet-stream"
-    blob_file = io.BytesIO()
-    blob.readinto(blob_file)
-    blob_file.seek(0)
-    return send_file(blob_file, mimetype=mime_type, as_attachment=False, download_name=path)
-    
-@app.route("/ask", methods=["POST"])
-def ask():
-    ensure_openai_token()
-    if not request.json:
-        return jsonify({"error": "request must be json"}), 400
-    approach = request.json["approach"]
-    try:
-        impl = ask_approaches.get(approach)
-        if not impl:
-            return jsonify({"error": "unknown approach"}), 400
-        r = impl.run(request.json["question"], request.json.get("overrides") or {})
-        return jsonify(r)
-    except Exception as e:
-        logging.exception("Exception in /ask")
-        return jsonify({"error": str(e)}), 500
-    
-@app.route("/chat", methods=["POST"])
-def chat():
-    ensure_openai_token()
-    if not request.json:
-        return jsonify({"error": "request must be json"}), 400
-    approach = request.json["approach"]
-    try:
-        impl = chat_approaches.get(approach)
-        if not impl:
-            return jsonify({"error": "unknown approach"}), 400
-        r = impl.run(request.json["history"], request.json.get("overrides") or {})
-        return jsonify(r)
-    except Exception as e:
-        logging.exception("Exception in /chat")
-        return jsonify({"error": str(e)}), 500
+        abort(404)  
```

Comment:
can you avoid the whitespace changes on this diff? (i think all these lines are just whitespace changes)